modules/core/src/lib/init.js

```diff
@@ -40,13 +40,10 @@ if (existingVersion && existingVersion !== version) {
 }
 
 if (!existingVersion) {
-  // eslint-disable-next-line
-  if (process.env.NODE_ENV !== 'production') {
-    log.log(
-      0,
-      `deck.gl ${version} - set deck.log.level=1 (or higher) to trace attribute updates`
-    )();
-  }
+  log.log(
```

Comment:
Looks good now.

Nit: This could be a one-liner, so wonder if this will pass the formatting check (`prettier-check`) in the linter. If is doesn't, running `yarn lint fix` should take care of it.